source/playlunky/mod/string_merge.cpp

```diff
@@ -19,7 +19,7 @@ bool StringMerger::RegisterOutdatedStringTable(std::string_view table)
     {
         return false;
     }
-    if (string_table > 8)
+    if (string_table > 12)
```

Comment:
should be 13 right?

Reply:
Is there string13.str file? I didn't find it in Spellunky 2\Mods\Extracted

Reply:
It's 0 indexed

Reply:
There is strings00 .. strings12 so string_table over 12 is invalid
checked value of string_table is 12 when file is strings12.str

Reply:
I'm going off of the original diff where it was >8 with strings07. will wait for malacath to comment though as i'm less familiar with this code

Reply:
Looks to me like there was a bug in the old code. I won't be able to test this until tomorrow unfortunately, but it looks sane to me. If any of you wanna rest strings12 we can merge also today.